helm/polaris/Chart.yaml

```diff
@@ -29,3 +29,6 @@ sources:
 keywords:
   - polaris
   - iceberg
+maintainers:
```

Comment:
This list is the result of `git shortlog -n -s -- helm`. It must contain valid GitHub handles.